README.md

```diff
@@ -62,6 +63,16 @@ You need the following permissions to run this module.
 - VPC Infrastructure Services
     - `Editor` role access
 
+<!-- END MODULE HOOK -->
+<!-- BEGIN EXAMPLES HOOK -->
+## Examples
+
+- [ Pre-wired CBR configuration for FS Cloud example](examples/fscloud)
+- [ Multi resource rule example](examples/multi-resource-rule)
+- [ CBR multi service profile](examples/multi-service-profile)
+- [ Multi-zone example](examples/multizone-rule)
+- [ Zone example](examples/zone)
+<!-- END EXAMPLES HOOK -->
```

Comment:
This hook is depreciated as examples are part of Overview hook. Remove it from here